macros/edr/system/configuration/get_configured_schemas_from_graph.sql

```diff
@@ -11,12 +11,9 @@
                     {% for node in depends_on_nodes %}
                         {% set node_package_name = node.get('package_name') %}
                         {% if node_package_name != 'elementary' %}
-                            {% if adapter.check_schema_exists(node.database, node.schema) %}
-                                    {% set schema_relation = api.Relation.create(database=node.database, schema=node.schema).without_identifier() %}
-                                    {% if schema_relation %}
-                                        {% set full_schema_name = schema_relation.render() %}
-                                        {% do configured_schemas.append(full_schema_name | upper) %}
-                                    {% endif %}
+                            {% if adapter.check_schema_exists(node['database'], node['schema']) %}
+                                {% set schema_relation = api.Relation.create(database=node.database, schema=node.schema).without_identifier() %}
+                                {% do configured_schemas.append((node['database'], node['schema'], schema_relation)) %}
```

Comment:
I saw in dbt that they return a tupple like a list so do it this way - append([node['database'], node['schema']])